modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationForm.testcase

```diff
@@ -82,7 +82,7 @@ definition {
 			EVPUtils.viewSuccessMessage();
 		}
 
-		task ("Then the employee user view a Request Created and also view Organization Created") {
+		task ("Then the Employee user view a Request Created and also view Organization Created") {
```

Comment:
Just a typo here, could you change this task to: `"Then the Employee user can see the Request  and Organization Created on listing"`